src/controllers/users.ts

```diff
@@ -0,0 +1,32 @@
+import User from '../models/user';
+import { CreateUserParams } from '../interfaces';
+
+export const getUsers = async () => {
+  const users = await User.findAll();
+  return users;
+};
```

Comment:
It seems unnecessary here to declare the variable users and then returning it. I think the following may be better 
```suggestion
export const getUsers = async () => await User.findAll();
```